### PR TITLE
feat: separate page for tooltips

### PR DESCRIPTION
**Pull request by ElviraMustafina**
Added separate page for desktop tooltips + new gif

note: the topic is temporarily placed top-level inside "Compose Multiplatform UI framework", I'll move it under "Desktop-specific components" when other parts are ready

### CODE REVIEW

**Comment by igordmn** [comment]
@m-sasha, could you review it for correctness?

File: topics/compose/compose-desktop-tooltips.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                                text = "Tooltip for ${name}",
+                                modifier = Modifier.padding(10.dp)
+                            )
+                        }
+                    },
+                    modifier = Modifier.padding(start = 40.dp),
+                    delayMillis = 600, // In milliseconds
+                    tooltipPlacement = TooltipPlacement.CursorPoint(
+                        alignment = Alignment.BottomEnd,
+                        offset = if (index % 2 == 0) DpOffset(-16.dp, 0.dp) else DpOffset.Zero // Tooltip offset
```
Comment (m-sasha) [inline topics/compose/compose-desktop-tooltips.md:57]: IDEA recomments `(-16).dp`

File: topics/compose/compose-desktop-tooltips.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+{initial-collapse-state="collapsed" collapsed-title="TooltipArea(tooltip = { Surface( "}
+
+<img src="compose-desktop-tooltips.png" alt="Tooltips" width="288" animated="true"/>
+
+The main parameters of the `TooltipArea` composable are:
+
+* `tooltip`, composable content for a tooltip.
+* `tooltipPlacement`, defines the tooltip placement. You can specify an anchor (the mouse cursor or the component),
+  an offset, and an alignment.
+* `delay`, the time in milliseconds after which the tooltip is shown. The default value is 500 ms.
```
Comment (m-sasha) [inline topics/compose/compose-desktop-tooltips.md:76]: It's `delayMillis`.

**Review by m-sasha — APPROVED** [review]

File: topics/compose/compose-desktop-tooltips.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+The main parameters of the `TooltipArea` composable are:
+
+* `tooltip`, composable content for a tooltip.
+* `tooltipPlacement`, defines the tooltip placement. You can specify an anchor (the mouse cursor or the component),
+  an offset, and an alignment.
+* `delayMillis`, the time in milliseconds after which the tooltip is shown. The default value is 500 ms.
+
+## What's next?
+
+You can also explore the tutorials about [other desktop components](https://github.com/JetBrains/compose-multiplatform/tree/master/tutorials#desktop).
```
Comment (koshachy) [inline topics/compose/compose-desktop-tooltips.md:79]:
```suggestion
Explore the tutorials about [other desktop components](https://github.com/JetBrains/compose-multiplatform/tree/master/tutorials#desktop).
```

**Review by koshachy — APPROVED** [review]
LGTM!

**Pull request merged**